lib/usd/ui/layerEditor/layerTreeItem.cpp

```diff
@@ -370,7 +375,14 @@ void LayerTreeItem::saveAnonymousLayer()
 
             // now replace the layer in the parent
             if (isRootLayer()) {
-                sessionState->rootLayerPathChanged(fileName);
+                const bool makePathRelative
+                    = MGlobal::optionVarExists(MAKE_PATH_RELATIVE_TO_SCENE_FILE)
+                    && MGlobal::optionVarIntValue(MAKE_PATH_RELATIVE_TO_SCENE_FILE);
+
```

Comment:
I think we may need to verify this in more places in the future so it may be good to provide a helper function to retrieve the option. (It would also make the code  support the possibility that we have a prefs or a UI for the option in the future.)